includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -74,13 +110,22 @@ public function enqueue_assets( $hook_suffix ) {
 			true
 		);
 
+		wp_enqueue_style(
+			self::JS_HANDLE,
+			amp_get_asset_url( "css/amp-setup-compiled{$rtl}.css" ),
+			[],
+			AMP__VERSION
+		);
```

Comment:
WordPress can decide whether to load the RTL version or not. Also, you could use the same version as the JS asset above to aid with cache busting:

```suggestion
		wp_enqueue_style(
			self::JS_HANDLE,
			amp_get_asset_url( 'css/amp-setup-compiled.css' ),
			[],
			$version
		);
		wp_styles()->add_data( self::JS_HANDLE, 'rtl', 'replace' );
```